docs/customization.md

```diff
@@ -263,15 +264,6 @@ that you need to be a member of [Insiders] to access its repository.
 
     [as described in the Insiders documentation]: insiders/getting-started.md#requirements
 
-=== "Insiders (with SSH key)"
```

Comment:
I meant to put that into a code annotation, similar to. Sorry if this was unclear. It's definitely valuable information and we should mention it, but I'd regard it as side content to the interested reader:
https://squidfunk.github.io/mkdocs-material/getting-started/#with-pip-9x

Reply:
Ah, sorry, you mean like so?
```
=== "Insiders"
    
    You will need to have a GitHub access token [as described in the
    Insiders documentation] and make it available in the `$GH_TOKEN`
    variable. 
  
    ``` sh
    git clone https://${GH_TOKEN}@github.com/squidfunk/mkdocs-material-insiders.git # (1)!
    ```

    1. If you are using SSH keys for authenticating with GitHub, you can
       clone Insiders with this command:
       ```
       git clone git@github.com:squidfunk/mkdocs-material-insiders.git
       ``` 
       
    [as described in the Insiders documentation]: insiders/getting-started.md#requirements
```

I had to remove the line before the code snippet in the annotation to force it to be rendered inside the annotation and not as a next line. The [documentation for the code annotation feature](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#adding-annotations) does not say anything about the rules for defining the annotation? Is this down to indentation? (may need to better learn Markdown's rules on that)

Reply:
I've gone and pushed this version, hope it fits the bill.

Reply:
> I had to remove the line before the code snippet in the annotation to force it to be rendered inside the annotation and not as a next line. The [documentation for the code annotation feature](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#adding-annotations) does not say anything about the rules for defining the annotation?

I think this happens because you need to indent by four spaces, not three, like so:

https://github.com/squidfunk/mkdocs-material/blob/5ba794405ff2781d57899d85eba3b0cda0e553c6/docs/getting-started.md?plain=1#L33-L55